app/routes.js

```diff
@@ -36,7 +37,8 @@ export const routes = (authProvider) => {
   }));
 
   router.get('/organisation/neworder', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
-    res.status(200).send('new order page');
+    const context = getNewOrderPageContext();
+    res.render('pages/order-dashboard/template.njk', addContext({ context, user: req.user }));
```

Comment:
is it the order dashboard page or is it the task list page? (too many dashboards)

Reply:
Glad you brought this up. I wasn't too comfortable with the name. I agree too many dashboards is getting confusing. specially when both dashboards are around orders

Shall we go for `order-tasks-list` instead?

Reply:
`order-task-list`

Reply:
lol yeah